src/Abp.AspNetCore/AspNetCore/Mvc/ExceptionHandling/AbpExceptionPageFilter.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Abp.AspNetCore.Configuration;
+using Abp.AspNetCore.Mvc.Results;
+using Abp.Authorization;
+using Abp.Dependency;
+using Abp.Domain.Entities;
+using Abp.Events.Bus;
+using Abp.Events.Bus.Exceptions;
+using Abp.Logging;
+using Abp.Reflection;
+using Abp.Runtime;
+using Abp.Runtime.Validation;
+using Abp.Web.Models;
+using Castle.Core.Logging;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Abp.AspNetCore.Mvc.ExceptionHandling
+{
+    public class AbpExceptionPageFilter : IAsyncPageFilter, ITransientDependency
+    {
+        public ILogger Logger { get; set; }
+
+        public IEventBus EventBus { get; set; }
+
+        private readonly IErrorInfoBuilder _errorInfoBuilder;
+        private readonly IAbpAspNetCoreConfiguration _configuration;
+
+        public AbpExceptionPageFilter(IErrorInfoBuilder errorInfoBuilder, IAbpAspNetCoreConfiguration configuration)
+        {
+            _errorInfoBuilder = errorInfoBuilder;
+            _configuration = configuration;
+
+            Logger = NullLogger.Instance;
+            EventBus = NullEventBus.Instance;
+        }
+
+        public async Task OnPageHandlerSelectionAsync(PageHandlerSelectedContext context)
```

Comment:
how about just return the completed task?
```suggestion
        public Task OnPageHandlerSelectionAsync(PageHandlerSelectedContext context)
        {
            return Task.CompletedTask;
        }
```

Reply:
In fact, this is the code I copied.

https://github.com/aspnetboilerplate/aspnetboilerplate/blob/dev/src/Abp.AspNetCore/AspNetCore/Mvc/Auditing/AbpAuditPageFilter.cs#L24